src/components/UI/WishList/List/List.jsx

```diff
@@ -0,0 +1,59 @@
+const List = ({ WishListItems: { wishlistItems, cartDispatch } }) => {
+  return (
+    <>
+      {wishlistItems &&
```

Comment:
change: not sure if this works but check with wislistItems.length

Reply:
yea it is working fine as expected you can review it.